src/SFA.DAS.ProviderApprenticeshipsService.Web.UnitTests/Orchestrators/Commitments/WhenGettingCommitmentDetails.cs

```diff
@@ -204,7 +205,7 @@ public async Task AndApprenticeshipIsOverFundingLimitThenACostWarningShouldBeAdd
                             },
                             EffectiveFrom = new DateTime(2020, 2, 1),
                             EffectiveTo = new DateTime(2020, 3, 1),
-                            Title = "Tit"
+                            Name = "Tit"
```

Comment:
This name made a lot more sense when the variable was Title

Reply:
The result was for it to combine the title and level to make the name variable - now this is being done in the API so name is appropriate